syntaxes/ballerina.YAML-tmLanguage

```diff
@@ -430,7 +430,7 @@ repository:
       - include: '#documentationDef'
       - include: '#functionDefinition'
       - include: '#annotationAttachment'
-      - match: (public|private|extern|resource|remote)
+      - match: (public|private|extern|resource|remote|isolated)
```

Comment:
I think we can remove extern here AFAIR it is not a keyword now

Reply:
Keeping it now as it's still used in 1.2.x.